packages/bolt-connection/test/connection-provider/connection-provider-direct.test.js

```diff
@@ -205,7 +205,7 @@ it('should call authenticationAuthProvider.handleError when TokenExpired happens
   expect(handleError).toBeCalledWith({ connection: conn, code: 'Neo.ClientError.Security.TokenExpired' })
 })
 
-it('should change error to retriable when error when TokenExpired happens and staticAuthTokenManager is not being used', async () => {
+it('should change error to retriyable when error when TokenExpired happens and staticAuthTokenManager is not being used', async () => {
```

Comment:
Best of both worlds, eh? :smile: 

```suggestion
it('should change error to retryable when error when TokenExpired happens and staticAuthTokenManager is not being used', async () => {
```